anvil/distributions.py

```diff
@@ -82,37 +85,41 @@ class PhiFourScalar:
     -----
     The general form of the action is
 
-        S(\phi) = \sum_{x \in \Lambda} [
-
-            C_ising * \sum_{\mu = 1}^d \phi(x + e_\mu) \phi(x)
-
-          + C_quadratic * \phi(x)^2
+    .. math::
 
-          + C_quartic * \phi(x)^4
-
-        ]
-
-    where C_ising, C_quadratic and C_quartic are coefficients built from the two couplings
-    provided in the constructor, \Lambda is the lattice, d is the number of space-time
-    dimensions and and e_\mu is a unit vector in the \mu-th dimensions.
+        S(\phi) = \sum_{x \in \Lambda} \left[
+            C_{\rm ising} * \sum_{\mu = 1}^d \phi(x + e_\mu) \phi(x) +
+            C_{\rm quadratic} * \phi(x)^2 +
+            C_{\rm quartic} * \phi(x)^4
+        \right]
 
+    where :math:`C_{\rm ising}`, :math:`C_{\rm quadratic}` and
+    :math:`C_{\rm quartic}` are coefficients built from the two couplings
+    provided in the constructor, :math:`\Lambda` is the space-time lattice
+    (the sum over the lattice is a sum over the lattice sites),
+    d is the number of space-time dimensions and
+    :math:`e_\mu` is a unit vector in the :math:`\mu^{th}` dimension.
 
     Examples
     --------
     Consider the toy example of this class acting on a random state
 
+    >>> from anvil.geometry import Geometry2D
+    >>> from anvil.distributions import PhiFourScalar
+    >>> import torch
     >>> geom = Geometry2D(2)
-    >>> action = PhiFourAction(geom, "standard", {"m_sq": 4, "g": 0})
-    >>> state = torch.rand((1, 2*2))
-    >>> action(state)
+    >>> action = PhiFourScalar.from_standard(geom, **{"m_sq": 4, "g": 0})
+    >>> state = torch.rand((1, 2*2)) # 2-D so lattice cardinality is 4
+    >>> action.log_density(state)
```

Comment:
I know it's pretty anal but personally I'd replace `action` with `target` or something similar, just because `action.log_density` is a bit confusing.